homeassistant/components/somfy/.translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "abort": {
+            "already_setup": "You can only configure one Somfy account.",
+            "authorize_url_fail": "Unknown error generating an authorize url.",
+            "authorize_url_timeout": "Timeout generating authorize url.",
+            "external_setup": "Somfy successfully configured from another flow.",
+            "no_flows": "You need to configure Somfy before being able to authenticate with it. [Please read the instructions](https://www.home-assistant.io/components/somfy/)."
```

Comment:
You can't use markdown.

Reply:
So no link possible?

Reply:
I think you still can have a link. You just can't use markdown formatting.

Reply:
@balloob does not want HTML too.